.circleci/config.yml

```diff
@@ -422,31 +403,40 @@ on-integ-and-version-tags: &on-integ-and-version-tags
       only: /^v[0-9].*/
 
 
-platform-build-defs: &platform-build-defs
+after-linter: &after-linter
   requires:
-    - build-debian
-  <<: *on-integ-and-version-tags
-  # <<: *on-any-branch # debugging
+    - lint
+
+after-build-and-test: &after-build-and-test
+  requires:
+    - build-and-test
 
 after-platform-builds: &after-platform-builds
   requires:
     # - build-centos7
     - build-bionic
     - build-xenial
 
-after-linter: &after-linter
-  requires:
-    - lint
+platform-build-defs: &platform-build-defs
+  <<: *after-build-and-test
+  <<: *on-integ-and-version-tags
+  # <<: *on-any-branch # debugging
+
 
 workflows:
   version: 2
   build_and_package:
     jobs:
       - lint:
           <<: *on-any-branch
-      - build-debian:
+      - build-and-test:
           <<: *on-any-branch
           <<: *after-linter
+      - dockerhub-build:
+          name: dockerhub-edge
+          build_args: EDGE=1
+          <<: *after-build-and-test
+          <<: *on-any-branch
```

Comment:
```suggestion
          <<: *on-master
```

Reply:
(1) this is for testing the PR. will change to "on-master" once it is working.
(2) true. will add support for `LATEST_BRANCH=1.2`